search-app/src/cmr/search/api/request_context_user_augmenter.clj

```diff
@@ -38,12 +41,11 @@
   (let [{:keys [token]} context]
     (let [sids (cache/get-value (cache/context->cache context token-sid-cache-name)
                                 token
-                                #(or (sids-retriever/get-sids context token)
-                                     {:sids (acl/context->sids context)}))
-          token-guid (:guid sids)
```

Comment:
This breaks our log-miner code which tries to use the same token for workload queries that was used in prod. I'm not sure of another way to do this without logging the actual token which we don't want to do. Maybe we could log the token, but have some way of telling Splunk not to show it?